local/bin/06-privilege-org-space

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+ORG_NAME="demo-org"
+SPACE_NAME="demo-space"
+
+# get the host name "pcf/BINDING_ID", and then remove the pcf policy prefix
+# when we load the entitlement directly into the pcf policy,
+# the pcf policy prefix is redundant
+#APP_HOST_NAME=$(cf env $APP_NAME | grep authn_login | awk '{print $NF}' | sed 's/host\///g; s/"//g; s/,$//g')
+#APP_HOST_NAME=$(echo $APP_HOST_NAME | sed 's/pcf\///g')
+
+ORG_GUID=$(cf org --guid "$ORG_NAME")
+SPACE_GUID=$(cf space --guid "$SPACE_NAME")
+
+ORG_LAYER_NAME="$ORG_GUID"
+SPACE_LAYER_NAME="$ORG_GUID/$SPACE_GUID"
+
+echo $ORG_LAYER_NAME
+echo $SPACE_LAYER_NAME
```

Comment:
Should we instead print out something like `"org name: $ORG_LAYER_NAME"`. It'll be pretty obvious what's being printed with the names currently hardcoded at the top of the file, but if those ever change it won't.